includes/class-scd-ext-dependency-checker.php

```diff
@@ -19,13 +20,21 @@ class Scd_Ext_Dependency_Checker {
 	 */
 	public static function are_system_dependencies_met() {
 		$are_met = true;
+
 		if ( ! self::check_php() ) {
 			add_action( 'admin_notices', array( __CLASS__, 'add_php_notice' ) );
 			$are_met = false;
 		}
+
+		// WordPress check is soft requirement for now.
+		if ( ! self::check_wp() ) {
+			add_action( 'admin_notices', array( __CLASS__, 'add_wp_notice' ) );
```

Comment:
For now I'm making this a soft requirement. Timezone stuff won't work _well_, but it shouldn't break with WP 5.3.